documentation-site/components/yard/index.tsx

```diff
@@ -23,16 +23,18 @@ const YardWrapper: React.FC<
       <Yard
         minHeight={placeholderHeight}
         pathname={router.pathname}
-        urlCode={router.query.code}
-        placeholderElement={() => {
+        urlCode={
+          typeof queryStringName !== 'undefined'
+            ? router.query[queryStringName]
+            : router.query.code
+        }
+        queryStringName={queryStringName}
+        placeholderElement={() => (
           if (!placeholderHeight) return null;
-
-          return (
```

Comment:
does this `return`  disappeared?

Reply:
just fixed it - i didn't resolve this merge conflict properly 🤦‍♂